Makefile

```diff
@@ -1,12 +1,25 @@
+bootstrap:
+	make clean-all
+	yarn
+	./node_modules/.bin/lerna bootstrap
+	make build
+
 build:
 	./scripts/build.sh
 
 publish: build
-	lerna publish
+	./node_modules/.bin/lerna publish
```

Comment:
were those changes needed?

Reply:
Yes, I had the .bin from node_modules inside my PATH by default, but if you don't have it the command will be unknown.

Changed in https://github.com/xtuc/charcodes/commit/c08d3c5906475f0e2e7b7d2b316c8abdef526e02